gallery_dl/extractor/bilibili.py

```diff
@@ -81,6 +81,31 @@ def items(self):
             yield Message.Url, url, text.nameext_from_url(url, article)
 
 
+class BilibiliUserArticleFavlistExtractor(BilibiliExtractor):
+    subcategory = "user-article-favlist"
```

Comment:
```suggestion
class BilibiliUserArticlesFavoriteExtractor(BilibiliExtractor):
    subcategory = "user-articles-favorite"
```

To stay consistent with the naming of the existing `user-articles` extractor and other `favorite` extractors